src/pages/new-claim/confirmation.tsx

```diff
@@ -0,0 +1,43 @@
+import { utils } from 'ethers';
+import Button from '../../components/button';
+import { Acknowledgement, FormState } from './new-claim-form';
+import { Policy } from '../../chain-data';
+import globalStyles from '../../styles/global-styles.module.scss';
+import styles from './new-claim-form.module.scss';
+
+interface Props {
+  form: FormState;
+  policy: Policy;
+  onSubmit: () => void;
+  onCancel: () => void;
+}
+
+export default function Confirmation(props: Props) {
+  const { form, policy } = props;
+
+  return (
+    <div className={styles.container}>
+      <ol className={styles.fieldList}>
+        <li>
+          <p>Enter the IPFS hash to your Claim Evidence form</p>
+          <p className={globalStyles.secondaryColor}>You created this hash in the previous step</p>
+          <p className={styles.confirmValue}>{form.evidence}</p>
+        </li>
+        <li>
+          <p>Requested relief amount, in USD</p>
+          <p className={globalStyles.secondaryColor}>
+            How much USD do you wish to receive? (Max of ${utils.commify(policy.coverageAmount.toString())})
```

Comment:
I wonder, when is the user informed about the payment actually being in API3?

Reply:
There will be a claim process document that we point to on the "Thank you for submitting" section, which will document all the user needs to know. There is also a new requirement that we will show both the USD and API token amount on the Claims Details page when paid out (which should make it clear)